tools/site_admin/show_common_words_from_project_word_lists.php

```diff
@@ -89,7 +89,7 @@
 
     echo "<tr>";
     echo "<td>" . _("Percentage cutoff:") . "</td>";
-    echo "<td><input type='text' name='cutoff' value='$cutoff' size='4'>% - ";
+    echo "<td><input type='number' style='width: 4em;' name='cutoff' value='$cutoff'>% - ";
```

Comment:
Can't access this page, might be a good idea to add `min=0` and `required`.

Reply:
Added `min=0`